src/index.js

```diff
@@ -95,6 +95,7 @@ async function fetchDocuments(ow, params) {
   const docs = [];
   const doc = {
     objectID: `${branch}--${path}`,
+    creationDate: new Date().getTime(),
```

Comment:
I'm sure that there is a way to automatically get the creation date of an index entry.

@olyver do you know ?

Reply:
Unfortunately there is no way for Algolia to automatically add the creation date or update date, it needs to be added from the code.